db/db_impl/db_impl.cc

```diff
@@ -407,7 +407,7 @@ Status DBImpl::ResumeImpl(DBRecoverContext context) {
     // during previous error handling.
     if (file_deletion_disabled) {
       // Always return ok
-      EnableFileDeletions(/*force=*/true);
+      s = EnableFileDeletions(/*force=*/true);
```

Comment:
Is the "Always return ok" still valid? If so, you might want to PermitUncheckedError here rather than status

Reply:
Sure.